packages/shared/src/storage/AppleKeyChainStorage/AppleKeyChainStorage.ts

```
@@ -0,0 +1,65 @@
+
+import type {
+  IAppleKeyChainNativeModule,
+  IAppleKeyChainStorage,
+} from './types';
+
+export class AppleKeyChainStorage implements IAppleKeyChainStorage {
+  private getKeychainModule(): IAppleKeyChainNativeModule {
+    if (platformEnv.isNativeIOS) {
+      const m = NativeModules?.KeychainModule;
+      if (!m) {
+        throw new OneKeyLocalError('Keychain native module not found');
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)